doc(release): Release notes for Cloud 24.07 (#3466)
centreon/centreon-documentation: cloud/releases/cloud-release-notes.md

title: Centreon Cloud release notes
---

> ## August xx, 2024

Reviewer: ```suggestion
## August 22, 2024
```

- Whitelist errors now have a ERROR level in Gorgone logs.

</details>

## Centreon map

### 24.07.0

Release date: `August 06 , 2024`

<details open>
  <summary>Enhancements</summary>

- [Editor] Added option to not display label for Containers.
- Display a message on the MAP widget when displayed in a public page.
- [GeoViews] Redesigned the edit modal.
- [VIEWER] Added possibility to display status as bg color for resource in weather style.

</details>

<details>
  <summary>Bug fixes</summary>

- [Map list] Fixed an issue that could prevent the map list from being displayed.
- Status of BA is not aligned with the status in Resource Status.

</details>

## Centreon mbi

### 24.07.0

Release date: `August 06 , 2024`

<details>
  <summary>Bug fixes</summary>

- - Fixed the perfdataStatisticsBuilder.pl script: when using the '-r --centile-only' parameters, the mod_bi_metrichourlyvalue table is no longer truncated.

</details>

## Centreon BAM

### 24.07.0

Release date: `August 06 , 2024`

<details>
  <summary>Bug fixes</summary>

- [BAM] Fixed Deprecation due to Symfony Upgrade.

</details>

> 

Reviewer 2: ## August xx, 2024

### Open Access to Centreon APIs

It is now possible to interact with Centreon through APIs and therefore integrate Centreon with any monitoring or data processing tool you may use, automatize monitoring configuration updates and any other use case you may have in mind. Access to APIs is granted thanks to tokens linked to contacts for access right management, but completely independent in terms of lifecycle.
API documentation is available here (link)

### Centreon Dashboards

It is now possible to declare dashboards playlists as public. An URL is genrated and give access to the dashboard playlist without authentication. This feature is ideal to display dahboards on big screens.
Note that this feature was implemented in a completely secured way. No confidential information is carried by the URL, nor it is possible to have access to any other information or data than the one displayed.
Note: MAP widget is not supported yet on public playlists.

This version of Centreon Dashboards comes with many other improvements. 
Graph widget allows much more configuration (automatic curves color adjustement for metric of same kind, bar chart and stacked bar chart presentation, 3 different curves design, and many others).
Widgets are also now more compact, allowing to display more of them on a single display.
And finally, freshness of data is now exposed and user can reload data at the widget level.

**Centreon Dashboards**

  It is now possible to declare dashboard playlists as public. A URL is generated and gives access to the dashboard playlist without authentication. This feature is ideal to display dashboards on big screens.
Note that this feature was implemented in a completely secured way. No confidential information is carried by the URL, nor it is possible to have access to any other information or data than the one displayed.
Note: MAP widget is not supported yet on public playlists.

This version of Centreon Dashboards comes with many other improvements.
Graph widget allows much more configuration (automatic curves color adjustment for metric of same kind, bar chart and stacked bar chart presentation, 3 different curves design, and many others).
Widgets are also now more compact, allowing to display more of them on a single display.
And finally, freshness of data is now exposed and user can reload data at the widget level.

**Miscellaneous improvements**

  Resource Status filter modal has been improved by adding resource type and host name filters.

### Centreon Business Edition

**MAP**

  The Geoview modal has been redesigned.

> In standard maps, it is now possible to hide container labels and to display status as background color for resources in weather style.

Reviewer: ```suggestion
In standard maps, it is now possible to hide container labels and to use the status as background color for resources in weather style.
```

## June 26, 2024

### Centreon IT Edition

**Resource Access Management**

>   Resource Access Management allows administrators to assign specific access permissions to each user or user group. This enables control over which users can access particular monitored resources. An intuitive interface helps manage these permissions, while audit logs provide a record of access activities. This feature supports organizations in maintaining security and efficiency by allowing tailored access control.

Reviewer: ```suggestion
Resource Access Management allows administrators to assign specific access permissions to each user or user group. This enables control over which users can access particular monitored resources. An intuitive interface helps manage these permissions, while audit logs provide a record of access activities. This feature supports organizations in maintaining security and efficiency by allowing tailored access control.
```
